driver/fillers_table.c

```diff
@@ -297,13 +297,20 @@ const struct ppm_event_entry g_ppm_events[PPM_EVENT_MAX] = {
 	[PPME_SYSCALL_LINK_2_X] = {FILLER_REF(sys_autofill), 3, APT_REG, {{AF_ID_RETVAL}, {0}, {1} } },
 	[PPME_SYSCALL_LINKAT_2_E] = {FILLER_REF(sys_empty)},
 	[PPME_SYSCALL_LINKAT_2_X] = {FILLER_REF(sys_linkat_x)},
-	[PPME_SYSCALL_FCHMODAT_E] = {FILLER_REF(sys_empty)},
-	[PPME_SYSCALL_FCHMODAT_X] = {FILLER_REF(sys_fchmodat_x)},
-	[PPME_SYSCALL_CHMOD_E] = {FILLER_REF(sys_empty)},
-	[PPME_SYSCALL_CHMOD_X] = {FILLER_REF(sys_chmod_x)},
-	[PPME_SYSCALL_FCHMOD_E] = {FILLER_REF(sys_empty)},
-	[PPME_SYSCALL_FCHMOD_X] = {FILLER_REF(sys_fchmod_x)},
-	[PPME_SYSCALL_RENAMEAT2_E] = {FILLER_REF(sys_empty)},
+	[PPME_SYSCALL_CHOWN_E] = {FILLER_REF(sys_autofill), 2, APT_REG, {{1}, {2} } },
```

Comment:
The order of the already present fillers here needs to be preserved (e.g fchmodat, fchmod) to keep scap compatibility.